Integration - Add support for Absolute API V3 (#38427)
demisto/content: Packs/Absolute/Integrations/Absolute/README.md

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| device_id | The system unique identifier of the device. | Required | 

> | limit | Maximum number of results to return. Default value is 50. Default is 50. | Optional | 

Reviewer: ```suggestion
| limit | Maximum number of results to return. Default is 50. | Optional | 
```

> | all_results | Whether to all results or not. Possible values are: true, false. | Optional | 

Reviewer: ```suggestion
| all_results | Whether to retrieve all results or not. Possible values are: true, false. | Optional | 
```

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| device_ids | A comma-separated list of device UIDs that should be unenrolled. | Required | 
| exclude_missing_devices | Whether to exclude missing devices or not. Possible values are: true, false. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |

> | Absolute.DeviceUnenroll.TotalDevices | String | The total devices in the request. | 

Reviewer: ```suggestion
| Absolute.DeviceUnenroll.TotalDevices | String | The total number of devices in the request. | 
```

- Using the or operator:
   - Get a list of all devices with less than 1 GB (1073741824 bytes) of available physical ram or less than 1 GB (1073741824 bytes) of available virtual raml: availablePhysicalMemroyBytes lt 1073741824 or availableVirtualMemoryBytes lt 1073741824

> For more examples and explanations, see the [Absolute docs](https://www.absolute.com/media/2221/abt-api-working-with-absolute.pdf) (from page 10).

Reviewer: Check this link.  It  did not work for me.

## Breaking changes from the previous version of this integration - Absolute
The following sections list the changes in this version.

### Arguments
#### The following arguments were removed in this version:

> In the *absolute-device-application-list* command:
> * *account_uids* - this argument was removed.

Reviewer: ```suggestion
In the ***absolute-device-application-list*** command, the following argument was removed:
* *account_uids*
```

### Outputs
- In the ***absolute-device-unenroll*** command the following outputs were replaced:
  * *Absolute.DeviceUnenroll.DeviceUid* - replaced by *Absolute.DeviceUnenroll.Devices.DeviceUid*.
  * *Absolute.DeviceUnenroll.ESN* - replaced by *Absolute.DeviceUnenroll.Devices.ESN*.

- In the ***absolute-device-freeze-request-get*** command the following outputs were replaced:
  * *Absolute.FreezeRequestDetail.Configuration.issuedUtc* - replaced by *Absolute.FreezeRequestDetail.Configuration.issuedDateTimeUTC*.
  * *Absolute.FreezeRequestDetail.RequesterUid* - replaced by *Absolute.FreezeRequestDetail.Requester*.
  * *Absolute.FreezeRequestDetail.Statuses.ackClientUtc* - replaced by *Absolute.FreezeRequestDetail.Statuses.ackClientDateTimeUtcs*.
  * *Absolute.FreezeRequestDetail.Statuses.scheduledFreezeDateUTC* - replaced by *Absolute.FreezeRequestDetail.Statuses.scheduledFreezeDateTimeUtc*.
  * *Absolute.FreezeRequestDetail.Statuses.updatedUtc* - replaced by *Absolute.FreezeRequestDetail.Statuses.updatedDateTimeUtc*.

> - In the ***absolute-device-application-list*** command the following outputs were removed:

Reviewer: ```suggestion
- In the ***absolute-device-application-list*** command the following output was removed:
```

  * *Absolute.DeviceApplication.AccountUid*

> - In the ***absolute-custom-device-field-list*** command the following outputs were removed:

Reviewer: ```suggestion
- In the ***absolute-custom-device-field-list*** command the following output was removed:
```
